_posts/2015-08-07-css-responsiveness.md

```diff
@@ -232,7 +232,7 @@ body{ }
 }
 {% endhighlight %}
 
-A desktop-first approach starts with styles for large screens and  `min-width` media queries in order to apply specific rules for larger viewports:
+A desktop-first approach starts with styles for large screens and  `max-width` media queries in order to apply specific rules for larger viewports:
```

Comment:
Thanks for noticing this. I guess `larger viewports` needs to be changed to `smaller viewports` then?